lib/native-client.js

```diff
@@ -118,94 +118,6 @@ class NativeClient extends EventEmitter {
   }
 
   /**
-   * Execute a buildInfo command on the currently connected instance.
-   *
-   * @param {Function} callback - The callback.
-   */
-  buildInfo(callback) {
-    this.database.admin().command({'buildInfo': 1}, {}, (error, result) => {
-      if (error) {
-        return callback(this._translateMessage(error));
-      }
-      callback(null, result);
-    });
-  }
-
-  /**
-   * Execute a hostInfo command on the currently connected instance.
-   *
-   * @param {Function} callback - The callback.
-   */
-  hostInfo(callback) {
-    this.database.admin().command({'hostInfo': 1}, {}, (error, result) => {
-      if (error) {
-        return callback(this._translateMessage(error));
-      }
-      callback(null, result);
-    });
-  }
-
-  /**
-   * Execute a connectionStatus command on the currently connected instance.
-   *
-   * @param {Function} callback - The callback.
-   */
-  connectionStatus(callback) {
```

Comment:
This method is publically available in DataService. Will need to verify Compass is not using it and remove it from the public API.